lib/pages/inventory.dart

```diff
@@ -1,8 +1,69 @@
 import 'package:flutter/material.dart';
+import 'package:flutter_redux/flutter_redux.dart';
+import 'package:scibble/models/product.dart';
+import 'package:scibble/redux/inventory/actions.dart';
+import 'package:scibble/redux/store.dart';
+import 'package:scibble/theme/scibble_color.dart';
+import 'package:scibble/widgets/hamburger.dart';
+import 'package:scibble/widgets/scibble_app_bar.dart';
 
 class Inventory extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
-    return Text('Inventar');
+    return Scaffold(
+        appBar: ScibbleAppBar(),
+        drawer: HamburgerMenu(),
+        body: StoreConnector<AppState, _InventoryViewModel>(
+          converter: (store) => _InventoryViewModel(store.state.inventory,
+              getInventory: store.dispatch(getInventory(store))),
+          builder: (_, viewModel) => InventoryViewModel(),
+          onInit: (store) => store.dispatch(getInventory(store)),
+        ));
+  }
+}
+
+class InventoryViewModel extends StatelessWidget {
+  final _InventoryViewModel viewModel;
+
+  InventoryViewModel({Key key, this.viewModel}) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return ListView.builder(
+      padding: EdgeInsets.all(10),
+      itemCount: viewModel._inventory.length,
```

Comment:
I am not sure if it can find `_inventory` as it is private, changing it to `inventory` to match the getter might work.